### ISSUE TITLE
Artemis crash my ps3 system

### ISSUE THREAD

**Issue by pftmclub** [bucanero/ArtemisPS3#10]
Hello, i had a trouble with Artemis ps3 and i need a help.
My ps3 model is CECH-2500A. Running Rebug lite 4.86.1 .
What i installed is Multiman, webman mod, irisman, psnpatch, SEN enabler and Artemis ps3. I'm running artemis, turn on double jump unlimited and max orb cheat for god of war 3. It's working with double jump, but after i'm going to teleport gate my ps3 goes black screen and then ask me to restart. I don't know what i did wrong, here is my picture
[](http://i.imgur.com/mBA0i9x.png)

**Comment by bucanero** [comment]
That kind of error is usually related to the PS3 running out of memory. Having too many loaded apps while running Artemis and a resource intensive game (like GOW3) can bring the ps3 to the limit.

Things to try:
- Switch the plugin version to `r5/HEN` (Go to Options menu, and change the plugin)
- unload webman and any other in-memory app, leave only Artemis and the game
- reduce webman memory settings

if you're loading an image, you can fully unload webman, and use Irisman to load the game.

(I never get issues with the r5 plugin, but I'm on 4.84)

**Comment by pftmclub** [comment]
> That kind of error is usually related to the PS3 running out of memory. Having too many loaded apps while running Artemis and a resource intensive game (like GOW3) can bring the ps3 to the limit.
> 
> Things to try:
> 
> * Switch the plugin version to `r5/HEN` (Go to Options menu, and change the plugin)
> * unload webman and any other in-memory app, leave only Artemis and the game
> * reduce webman memory settings
> 
> if you're loading an image, you can fully unload webman, and use Irisman to load the game.
> 
> (I never get issues with the r5 plugin, but I'm on 4.84)

So this is what i should did.
1. Turn on ps3
2.  Open Irisman , launch gow 3
3.  Open artemis and enable gow3 cheat.
4.  Choose the gow 3 launcher and then press start to attach .
5. Profit?

Correct me if its wrong, i don't get the unload webman thing

**Comment by bucanero** [comment]
yes, that would be ok, but first I'd suggest trying unloading webman mod from memory

if you go to the "webman" folder, in the "Setup" section, there's an option called "unload webman"
there's also a pad combination to unload webman but I don't remember it

it's not needed to uninstall webman, just to unload it from memory to leave all the available memory to Artemis and the game.

Also, don't forget to try the `r5` plugin in the Artemis options menu

**Comment by pftmclub** [comment]
> yes, that would be ok, but first I'd suggest trying unloading webman mod from memory
> 
> if you go to the "webman" folder, in the "Setup" section, there's an option called "unload webman"
> there's also a pad combination to unload webman but I don't remember it
> 
> it's not needed to uninstall webman, just to unload it from memory to leave all the available memory to Artemis and the game.
> 
> Also, don't forget to try the `r5` plugin in the Artemis options menu

After i unload webman from memory the ps3 fan start very loudy , it's seem uninstall webman is the only choice i had

**Comment by bucanero** [comment]
> After i unload webman from memory the ps3 fan start very loudy , it's seem uninstall webman is the only choice i had

That is related to webman fan-control settings. If you set it to `SYSCON` (controlled by the PS3) then you can unload webman without fan-noise issues

**Comment by stinger101mg** [comment]
Yeah cuz you unloaded the webman plugin but I don't think when you do that it correctly turns back control to syscon over the fan control, ergo why youre fan ramps up cuz it doesn't know what to do other than most likely a failsafe default fan speed in case syscon somehow lost fan control. However I'm merely speculating here but that's the only thing I can think it would be.

**Comment by goldug** [comment]
> > yes, that would be ok, but first I'd suggest trying unloading webman mod from memory
> > if you go to the "webman" folder, in the "Setup" section, there's an option called "unload webman"
> > there's also a pad combination to unload webman but I don't remember it
> > it's not needed to uninstall webman, just to unload it from memory to leave all the available memory to Artemis and the game.
> > Also, don't forget to try the `r5` plugin in the Artemis options menu
> 
> After i unload webman from memory the ps3 fan start very loudy , it's seem uninstall webman is the only choice i had

Yes, Webman controls fan speed normally.
And yes, on a lot of PS3's Webman takes too much memory so it can't run Artemis or CCAPI at the same time. Even setting Webman to the lowest memory usage won't suffice in most cases.
You don't have to completely uninstall Webman, you can just edit the boot_plugins.txt file. I have a homebrew that lets me choose which boot_plugins.txt to load, so you can change it on the fly (with a reboot inbetween).

**Comment by NickDeBeenSAE** [comment]
NFS Undercover cheats from my point of view just don't work.
